Google.Ads.Gax/src/Util/ProtobufUtilities.cs

```diff
@@ -83,6 +83,17 @@ public static U Parse<U>(byte[] data) where U : IMessage<U>, new()
             return parser.ParseFrom(data);
         }
 
+        /// <summary>
+        /// Parses the failure.
+        /// </summary>
+        /// <typeparam name="U"></typeparam>
+        /// <param name="message">The message.</param>
+        /// <returns></returns>
+        public static byte[] Serialize<U>(U message) where U : IMessage<U>, new()
```

Comment:
Is this method really necessary? It's just renaming the common protobuf `ToByteArray()` method.

Reply:
I keep finding myself googling for it every time because it is an extension method and not easy to locate each time. That's why I put it in a convenient place.